ExperimentSettings 	experiment
HomeSettings 		home
float64 			waitEntry
TriggerSettings 	triggerEntry
MoveSettings 		move
LasertrackSettings 	lasertrack
TriggerSettings 	triggerExit
SaveSettings 		save
---
bool 				succeeded

